sample_onsr/oci-objectstorage-onsr-put-object-go/func.go

```diff
@@ -0,0 +1,105 @@
+/*
+ oci-objectstorage-onsr-put-object-go version 1.0.
+
+ Copyright (c) 2021 Oracle, Inc.  All rights reserved.
+ Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+
+package main
+
+import (
+	"context"
+	//"crypto/tls"
+	//"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	//"io/ioutil"
+	"os"
+	"path"
+
+	fdk "github.com/fnproject/fdk-go"
+	"github.com/oracle/oci-go-sdk/v37/common"
+	"github.com/oracle/oci-go-sdk/v37/common/auth"
+	"github.com/oracle/oci-go-sdk/v37/example/helpers"
+	"github.com/oracle/oci-go-sdk/v37/objectstorage"
+)
+
+type ObjectStorage_Bucket struct {
+	Name string `json:"bucket"`
+}
+
+func main() {
+	fdk.Handle(fdk.HandlerFunc(myHandler))
+}
+
+func putObject(ctx context.Context, c objectstorage.ObjectStorageClient, namespace, bucketname, objectname string, contentLen int64, content io.ReadCloser, metadata map[string]string) error {
+	request := objectstorage.PutObjectRequest{
+		NamespaceName: common.String(namespace),
+		BucketName:    common.String(bucketname),
+		ObjectName:    common.String(objectname),
+		ContentLength: common.Int64(contentLen),
+		PutObjectBody: content,
+		OpcMeta:       metadata,
+	}
+	_, err := c.PutObject(ctx, request)
+	fmt.Println("put object")
```

Comment:
Can we modify this to add  details like object name and any other relevant info..also if the status was success of failed.